models.go

```diff
@@ -15,10 +15,12 @@ type Options struct {
 	Sort       []map[string]string
 	MaxRecords int
 	View       string
+	Offset     string
 }
 
 type records struct {
 	Records []Record `json:"records"`
+	Offset  string   `json:"offset"`
```

Comment:
Maybe add an `omitempty` here? Like so:

```go
type records struct {
   // ...
  Offset string `json:"offset,omitempty"`
}
```

Consequently, this means your test update [here](https://github.com/mottaquikarim/go-airtable/pull/11/files#diff-ea861308f9377d3cfd33952aaabf1a07R164) will have to be reverted.

I'm a +0 on this, generally prefer not to send more stuff over the wire than I have to but am open to your input and can be persuaded one way or another

Reply:
I agree with this, adding omitempty